prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -71,7 +71,7 @@ class FHIRRouter(
      *   At least one of patient street, patient zip code, patient phone number, patient email
      *   At least one of order test date, specimen collection date/time, test result date
      */
-    private val fullElrQualityFilterDefault: ReportStreamFilter = listOf(
+    val fullElrQualityFilterDefault: ReportStreamFilter = listOf(
```

Comment:
I think if we want to keep these around as examples, we should reframe them, instead of being "default", maybe "sample"? Also might be best to move these directly to the test files to make it clear that they aren't used in the code.